lib/engines/content_block_manager/app/components/content_block_manager/content_block/document/show/host_editions_table_component.rb

```diff
@@ -53,7 +53,8 @@ def frontend_path(content_item)
   end
 
   def content_link(content_item)
-    link_to(content_item.title, frontend_path(content_item), class: "govuk-link")
+    link_to(sanitize(content_item.title + tag.span(" (opens in new tab)", class: "govuk-visually-hidden")),
```

Comment:
I find this a bit hard to grok with all the brackets, maybe having a method called `content_link_text` with something like this would be better?

```ruby
def content_link_text(content_item)
  sanitize [
    content_item.title,
    tag.span("(opens in new tab)", class: "govuk-visually-hidden"),
  ].join(" ")
end
```

Reply:
good idea!